Makefile

```diff
@@ -47,6 +47,9 @@ PROJECT_ORG = $(PROJECT_TAG)
 PROJECT_URL = https://github.com/$(PROJECT_ORG)
 PROJECT_BUG_URL = $(PROJECT_URL)/kata-containers/issues/new
 
+# @RUNTIME_NAME@ should be replaced with the target in generated files
+RUNTIME_NAME = $(TARGET)
```

Comment:
@c3d, shouldn't `RUNTIME_NAME` be set only after `TARGET` is set?
TARGET is set a few lines below.

Reply:
Does not really matter if you assign it with `=`. In that case, it's re-evaluated when you use it.

Reply:
As you pointed out in the PR for 2.0.0, at the first look it's confusing, indeed.
There's no need to change it, unless you want to / will already rework the series for some other reason.

Thanks for the explanation!